test/forEach.js

```diff
@@ -154,6 +154,6 @@ test('first element errors', async (t) => {
     })
     t.fail('must throw')
   } catch (err) {
-    t.equal(started > 16, true)
+    t.equal(started === 16, true)
```

Comment:
This should be great then 16. Maybe we should update the test case to add some delay?

Reply:
I don't see how/why this should be greater than 16?

Reply:
We have 16 elements (up to the highwatermark) that should start processing, then all of them wait for an event loop run (`immediate()`), then the first errors.

Reply:
Yes, the highwatermark is reached and then one errors, why/when would the 17th be queued?

Reply:
Ah it's ok!